go/engine/revoke.go

```diff
@@ -139,6 +139,14 @@ func (e *RevokeEngine) run(m libkb.MetaContext) error {
 		hasPGP := len(me.GetComputedKeyFamily().GetActivePGPKeys(false)) > 0
 
 		if len(me.GetComputedKeyFamily().GetAllActiveDevices()) == 1 {
+			passphraseState, err := libkb.LoadPassphraseState(m)
```

Comment:
New logic serverside is to disallow revoking last device without a passphrase since then login-after-reset-with-passphrase is not possible.

Reply:
Does this mean we disallow self-revoking in e.g. a border-crossing situation with no internet where someone would rather brick their account than have it compromised?